monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -564,6 +569,7 @@
         },
         {
           alert: 'CephNodeDiskspaceWarning',
+          'for': '1m',
```

Comment:
predict linear, take the into account the history of the metric in the last two days.. Why we need to wait 1 minute to warn?